README.rst

```diff
@@ -0,0 +1,415 @@
+   :alt: FOSSA Status
+
+.. image:: https://bestpractices.coreinfrastructure.org/projects/2373/badge
+   :target: https://bestpractices.coreinfrastructure.org/projects/2373
+   :alt: CII Best Practices
+
+.. image:: https://pepy.tech/badge/horovod
+   :target: https://pepy.tech/project/horovod
+   :alt: Downloads
+
+.. inclusion-marker-start-do-not-remove
+
+.. contents::
+
+
+Horovod is a distributed training framework for TensorFlow, Keras, PyTorch, and MXNet. The goal of Horovod is to make
+distributed Deep Learning fast and easy to use.
+
+.. image:: https://github.com/LFDLFoundation/artwork/raw/master/lfdl/horizontal/color/lfdl-horizontal-color.png
+   :width: 200px
+   :alt: LF DL
+
+Horovod is hosted by the `Linux Foundation Deep Learning <https://lfdl.io>`_ (LF DL). If you are a company that is deeply
+committed to using open source technologies in artificial intelligence, machine and deep learning, and wanting to support
+the communities of open source projects in these domains, consider joining the LF Deep Learning Foundation. For details
+about who's involved and how Horovod plays a role, read the LF DL `announcement <https://lfdl.io/press/2018/12/13/lf-deep-learning-welcomes-horovod-distributed-training-framework-as-newest-project/>`_.
+
+
+Why not traditional Distributed TensorFlow?
+-------------------------------------------
+
+The primary motivation for this project is to make it easy to take a single-GPU TensorFlow program and successfully train
+it on many GPUs faster. This has two aspects:
+
+1. How much modification does one have to make to a program to make it distributed, and how easy is it to run it.
+2. How much faster would it run in distributed mode?
+
+Internally at Uber we found the MPI model to be much more straightforward and require far less code changes than the
+Distributed TensorFlow with parameter servers. See the [Usage](#usage) section for more details.
+
+In addition to being easy to use, Horovod is fast. Below is a chart representing the benchmark that was done on 128
+servers with 4 Pascal GPUs each connected by RoCE-capable 25 Gbit/s network:
+
+.. image:: https://user-images.githubusercontent.com/16640218/38965607-bf5c46ca-4332-11e8-895a-b9c137e86013.png
+   :alt: 512-GPU Benchmark
+
+Horovod achieves 90% scaling efficiency for both Inception V3 and ResNet-101, and 68% scaling efficiency for VGG-16.
+See the [Benchmarks](docs/benchmarks.md) page to find out how to reproduce these numbers.
+
+While installing MPI and NCCL itself may seem like an extra hassle, it only needs to be done once by the team dealing
+with infrastructure, while everyone else in the company who builds the models can enjoy the simplicity of training them at
+scale.
+
+
+Install
+------------
+
+To install Horovod:
+
+1. Install `Open MPI <https://www.open-mpi.org/>`_ or another MPI implementation.
+
+Steps to install Open MPI are listed `on this page <https://www.open-mpi.org/faq/?category=building#easy-build>`_.
```

Comment:
Learn how to install Open MPI on this page

Reply:
Also, recommend making this a second sentence after "another MPI implementation" (same line)